packages/contracts-ops/contracts/scripts/DeployImplementations.s.sol

```diff
@@ -114,5 +104,18 @@ contract DeployImplementations is DeployImplementationsLogic {
             _configPath,
             bridgeAttributePath(_domain, "tokenRegistry.implementation")
         );
+        __Config_reload(_configPath);
+    }
+}
+
+contract DeployImplementations is DeployImplementationsLogic {
+    // entrypoint
+    function deploy(string memory _configPath, string memory _domain) public {
+        __Config_initialize(_configPath);
+        vm.createSelectFork(getRpcs(_domain)[0]);
+        vm.startBroadcast();
+        deployImplementations(_domain);
+        vm.stopBroadcast();
+        updateImplementations(_domain, _configPath);
```

Comment:
same naming comment. this does not update the implementation, it records an update to the json

Reply:
https://github.com/nomad-xyz/monorepo/pull/511/commits/b758804fea4adf32e72569a32d4d3683320de04b